### ISSUE TITLE
Download-count update seems to be broken

### ISSUE THREAD

**Issue by BracketJohn** [nuxt/modules#510]
When checking out the modules page today, I saw that the download count for `nuxt-session` is off. It is correct on the modules details page

On the repo: https://github.com/sidebase/nuxt-session
![image](https://user-images.githubusercontent.com/25911230/203736026-c100e229-3c24-4224-a359-8e159489b332.png)

On the modules page: https://nuxt.com/modules
![image](https://user-images.githubusercontent.com/25911230/203735823-e77ddff2-3360-4ea6-af18-69a52d561214.png)

On the modules details page: https://nuxt.com/modules/session
![image](https://user-images.githubusercontent.com/25911230/203736156-3ba334c0-ecaf-4dbd-a9ff-054b78d0dba3.png)

In addition the maintainer-count seems to be broken (5 on overview, 1 on details page of `nuxt.com/modules`). 

So in summary:
- [ ] download counts between modules-overview and module-details is off
- [ ] maintainer counts between modules-overview and module-details is off

**Comment by danielroe** [comment]
We show the monthly download count, not downloads across all time.

![image.png](https://volta.s3.fr-par.scw.cloud/image_c5c9a3c613.png)

The maintainer count is coming from the data in this repository - so you should feel free to update it.

https://github.com/nuxt/modules/blob/3d6be08bffe66714c2a8fec873c3b60e1eb072b8/lib/modules.ts#L101-L114

**Comment by BracketJohn** [comment]
I see, thanks - maybe it would be worth it to update the text to `/ month`? 

Other than that my question has been answered and if we don't want to adapt it to say `/ month` the issue can be closed, thanks 🙌

**Comment by danielroe** [comment]
I think it might nice to be something like `x installs this month` in the 'fuller' detail page version, keeping the minimal approach on the card. wdyt?

**Comment by BracketJohn** [comment]
Yes, that would be great + maybe we can add a tooltip to the card-stats on hover, to also give that context there?
